eq-author-api/middleware/identification/verifyJwtToken.js

```diff
@@ -0,0 +1,32 @@
+const admin = require("firebase-admin");
+
+/*
+Must define firebase service account credentials for each environment:
+1. Download firebase service account json file and save in eq-author-api root as .firebase-admin-service-account.json
+    https://firebase.google.com/docs/admin/setup
+
+2. Set env variable. Create /eq-author-api/.env file containing:
+    GOOGLE_APPLICATION_CREDENTIALS=/app/.firebase-admin-service-account.json
```

Comment:
As discussed this can be hardcoded into the docker-compose no need for an env file

Reply:
Can it be added `.gitignore`